### PR TITLE
ci: test Node.js 6, 8, 10 and 11

### PR DESCRIPTION
**Pull request by DanielRuf**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/kvz/locutus/blob/master/CONTRIBUTING.md) document?
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/kvz/locutus/pulls) for the same update/change?

### Description
We should run tests on all current LTS branches which are 6, 8, 10 and current stable (11).

See https://github.com/nodejs/Release/blob/master/README.md

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,18 +7,17 @@ addons:
 
 language: node_js
 node_js:
-- '0.10' # these should be quoted; otherwise, 0.10 turns into 0.1
-- '0.12'
-- '4'
+- 'node' # these should be quoted; otherwise, 0.10 turns into 0.1
```
Comment (kvz) [inline .travis.yml]: Out of curiosity, what version is `node`?

File: .travis.yml [hunk 2]
```diff
@@ -7,18 +7,17 @@ addons:
...
 - '6' # When you update this, also update deploy.on.node further down, to tell who is the deployer build
-- '7'
 sudo: false
 matrix:
   allow_failures:
-  - node_js: '0.10'
-  - node_js: '0.12'
-  - node_js: '4'
-  - node_js: '7'
+  - node_js: 'node'
```
Comment (kvz) [inline .travis.yml]: Are all versions allowed to fail now?

File: .travis.yml (hunk 1 above)
Comment (DanielRuf) [reply .travis.yml]: `node`is current stable / latest which is currently 11.1.0.

File: .travis.yml (hunk 2 above)
Comment (DanielRuf) [reply .travis.yml]: As it was before.

Comment (DanielRuf) [reply .travis.yml]: I'll remove this part as all are green: https://travis-ci.org/kvz/locutus/builds/451812302

**Comment by kvz** [comment]
Okay thanks! Maybe we should add the latest stable as the allowed failure, since it moves and might break more easily. But we can always do that later 👌👍 thanks again!

**Comment by DanielRuf** [comment]
> Okay thanks! Maybe we should add the latest stable as the allowed failure, since it moves and might break more easily. But we can always do that later 👌👍 thanks again!

Exactly. Let's keep an eye on it (always check if the CI build is green or red - or the notifications from the new travis-ci.com platform) =)

**Pull request merged**